code-challenges/objects/challenges-06.test.js

```diff
@@ -111,6 +113,13 @@ The input and output of this function are the same as the input and output from
 
 const hasChildrenEntries = (arr, character) => {
   // Solution code here...
+  let children = 0;
+  Object.values(arr).forEach(person => {
```

Comment:
Object.values() is supposed to take in an object, not an array. If you pass an array in it will just return an array.

Object.values(arr).forEach() is ultimately the same as arr.forEach()